packages/app/src/elements/unlock.ts

```diff
@@ -215,6 +215,11 @@ export class Unlock extends StartForm {
             if (e.code !== ErrorCode.DECRYPTION_FAILED) {
                 throw e;
             }
+            if (this._failedCount > 3) {
+                await this.app.logout();
+                router.go("login");
+                return;
+            }
```

Comment:
Keeping the logout after 5 attempts is a great idea, since it's a simple change. I would add an alert though to inform people of what is happening.